src/components/progress/progress.scss

```diff
@@ -0,0 +1,117 @@
+@import "../../themes/ionic.globals";
+// Progress bar
+// --------------------------------------------------
+
+/// @prop - The color of the element that is being animated
+$progress-color:              rgb(16,108,200) !default;
+
+/// @prop - The background color of the progress bar
+$progress-buffer-color:       rgb(170,209,249) !default;
```

Comment:
I mean the entire variables alignment. This is how I see it (rgb is like 2 spaces before the others)
![image](https://user-images.githubusercontent.com/5757359/27773509-f080ae78-5f83-11e7-9930-a56f7be6dc18.png)